js/src/components/paid-ads/asset-group/asset-group-card.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useRef, useEffect, Fragment } from '@wordpress/element';
+import { ExternalLink } from 'extracted/@wordpress/components';
+import { SelectControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAdaptiveFormContext } from '.~/components/adaptive-form';
+import AppInputControl from '.~/components/app-input-control';
+import ImagesSelector from './images-selector';
+import TextsEditor from './texts-editor';
+import AssetField from './asset-field';
+import {
+	ASSET_IMAGE_SPECS,
+	ASSET_TEXT_SPECS,
+	ASSET_DISPLAY_URL_PATH_SPECS,
+} from '../constants';
+import './asset-group-card.scss';
+
+const ctaOptions = [
+	{ label: 'Automated', value: 'UNSPECIFIED' },
+	{ label: 'Learn more', value: 'LEARN_MORE' },
+	{ label: 'Get quote', value: 'GET_QUOTE' },
+	{ label: 'Apply now', value: 'APPLY_NOW' },
+	{ label: 'Sign up', value: 'SIGN_UP' },
+	{ label: 'Contact us', value: 'CONTACT_US' },
+	{ label: 'Subscribe', value: 'SUBSCRIBE' },
+	{ label: 'Download', value: 'DOWNLOAD' },
+	{ label: 'Book now', value: 'BOOK_NOW' },
+	{ label: 'Shop now', value: 'SHOP_NOW' },
+];
+
+/**
+ * Renders the Card UI for managing the an asset group.
+ *
+ * Please note that this component relies on an CampaignAssetsForm's context and custom adapter,
+ * so it expects a `CampaignAssetsForm` to existing in its parents.
```

Comment:
Since this is only rendered in `<AssetGroupSection>` I would remove this duplicated docs. I take as granted `<AssetGroupCard>` is not going to be used outside  `<AssetGroupSection>`.

Reply:
Seems like this is only used in `<AssetGroup>`. So IMO better only in `<AssetGroup>` to avoid duplication.

Reply:
Added this code comment is not based on the composition of components, but on whether a component uses `useAdaptiveFormContext` and the form adapter extended by `CampaignAssetsForm`.

It's independent and unrelated between components, so it's not a duplication but an indication of the component's dependency on the context provider.